app/controllers/search.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const User = mongoose.model('User');
+
+/**
+ * Gets all users from the database
+ */
+exports.users = (req, res) => {
+  const query = req.params.inviteeEmail || '';
+  User.find({ email: { $regex: query } }).limit(10)
```

Comment:
Why is the limit() function used? Is it possible to have more than one user with the same email?